File: Jimmy Selgen Nielsen/MessageAggregator/AirlineMessageSplitter - .NET/Messages/FlightDetailsInfoResponse.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Xml.Serialization;

namespace AirlineMessageSplitter.Messages
{
	[XmlRoot("FlightDetailsInfoResponse")]
	public class FlightDetailsInfoResponse
	{
		[XmlElement(ElementName = "Flight")]
		public Flight flight;
		[XmlElement(ElementName = "Passenger")]
		public Passenger passenger;
		[XmlElement(ElementName = "Luggage")]
		public List<Luggage> luggageList;
	}
}

File: Jimmy Selgen Nielsen/MessageAggregator/AirlineMessageSplitter - Java/src/main/java/com/example/airlinesplitter/messages/FlightDetailsInfoResponse.java
package com.example.airlinesplitter.messages;

import lombok.Data;
import lombok.extern.jackson.Jacksonized;

import jakarta.xml.bind.annotation.*;
import java.util.List;

@Data
@Jacksonized
@XmlRootElement(name = "root")
@XmlAccessorType(XmlAccessType.FIELD)
public class FlightDetailsInfoResponse {
    @XmlElement(name = "Flight")
    private Flight flight;
    @XmlElement(name = "Passenger")
    private Passenger passenger;
    @XmlElement(name = "Luggage")
    private List<Luggage> luggageList;
}
